lib360dataquality/cove/threesixtygiving.py

```diff
@@ -571,10 +571,10 @@ def get_heading_count(self, test_class_type):
             return self.count
 
         if self.aggregates["count"] == 1 and self.count == 1:
-            self.grants_percentage = 100
+            self.grants_percentage = 1.0
             return f"1 {self.relevant_grant_type}".strip()
 
-        if self.grants_percentage < 5:
+        if self.count < 5:
```

Comment:
Why change to `.count < 5` instead of `.grants_percentage < 0.05`?

Reply:
related to issue #93